runtime/help/commands.md

```diff
@@ -21,10 +21,13 @@ quotes here but these are not necessary when entering the command in micro.
    This command will modify `bindings.json` and overwrite any bindings to
    `key` that already exist.
 
-* `help ['topic']`: opens the corresponding help topic. If no topic is provided
-   opens the default help screen. Help topics are stored as `.md` files in the
-   `runtime/help` directory of the source tree, which is embedded in the final
-   binary.
+* `help ['topic'] ['flags']`: opens the corresponding help topic.
+   If no topic is provided opens the default help screen.
+   Help topics are stored as `.md` files in the `runtime/help` directory of
+   the source tree, which is embedded in the final binary.
+   The `flags` are optional.
+   * `-hsplit`: Opens the help topic in a `hsplit` (default for initial split)
```

Comment:
Maybe clearly say "in a horizontal split" (for those not yet familiar with the slang)?